src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -838,9 +851,13 @@ else if (level.equals("Error")) {
 
     private void setupTrayIcon() {
         mainFrame.addWindowListener(new WindowAdapter() {
+            @Override
             public void windowActivated(WindowEvent e)   { trayMenuMain.setLabel("Hide"); }
+            @Override
             public void windowDeactivated(WindowEvent e) { trayMenuMain.setLabel("Show"); }
+            @Override
             public void windowDeiconified(WindowEvent e) { trayMenuMain.setLabel("Hide"); }
+            @Override
```

Comment:
This is not my code. But in the same style as C style pragma this should probably be above the method.